packages/components/src/internal/components/editable/Cell.tsx

```diff
@@ -248,14 +254,19 @@ export class Cell extends React.PureComponent<Props, State> {
     handleSelect = (event): void => {
         const { cellActions, colIdx, rowIdx, selected } = this.props;
         const { selectCell } = cellActions;
+        const isDragHandle = event.target?.className?.indexOf(CELL_SELECTION_HANDLE_CLASSNAME) > -1;
```

Comment:
Move computation of `isDragHandle` inside the `else if`. This will make it so we don't process this every time `handleSelect` is fired.

Reply:
done